SQL/0000-00-00-schema.sql

```diff
@@ -688,20 +688,34 @@ CREATE TABLE `mri_protocol` (
   `Center_name` varchar(4) NOT NULL default '',
   `ScannerID` int(10) unsigned NOT NULL default '0',
   `Scan_type` int(10) unsigned NOT NULL default '0',
-  `TR_range` varchar(255) default NULL,
-  `TE_range` varchar(255) default NULL,
-  `TI_range` varchar(255) default NULL,
-  `slice_thickness_range` varchar(255) default NULL,
-  `FoV_x_range` varchar(255) default NULL,
-  `FoV_y_range` varchar(255) default NULL,
-  `FoV_z_range` varchar(255) default NULL,
-  `xspace_range` varchar(255) default NULL,
-  `yspace_range` varchar(255) default NULL,
-  `zspace_range` varchar(255) default NULL,
-  `xstep_range` varchar(255) default NULL,
-  `ystep_range` varchar(255) default NULL,
-  `zstep_range` varchar(255) default NULL,
-  `time_range` varchar(255) default NULL,
+  `TR_min` varchar(10) DEFAULT NULL,
+	`TR_max` varchar(10) DEFAULT NULL,
+	`TE_min` varchar(10) DEFAULT NULL,
+	`TE_max` varchar(10) DEFAULT NULL,
+	`TI_min` varchar(10) DEFAULT NULL,
+	`TI_max` varchar(10) DEFAULT NULL,
+	`slice_thickness_min` varchar(10) DEFAULT NULL,
+	`slice_thickness_max` varchar(10) DEFAULT NULL,
+	`FOV_x_min` varchar(5) DEFAULT NULL,
+	`FOV_x_max` varchar(5) DEFAULT NULL,
+	`FOV_y_min` varchar(5) DEFAULT NULL,
+	`FOV_y_max` varchar(5) DEFAULT NULL,
+	`FOV_z_min` varchar(5) DEFAULT NULL,
+	`FOV_z_max` varchar(5) DEFAULT NULL,
```

Comment:
I don't think FOV is used by the code at all and should just be removed, but ask @MounaSafiHarab and/or @cmadjar to be sure. I seem to recall it not being a real header when I tried to use them..

Reply:
Indeed, currently it does not look like the FOV are being used to identify a scan type in the [identify_scan_db](https://github.com/aces/Loris-MRI/blob/9cf1f5e0f8d6f315a0fb321cf228b6fb6fed6dd4/uploadNeuroDB/NeuroDB/MRI.pm#L394) function of the perl scripts. I also tried to grep for FOV in the LORIS-MRI code base and nothing came out.
Unless @MounaSafiHarab disagrees, it is probably safe to remove those FOV fields.

Reply:
I just checked PREVENT-AD and they have 6 files with this header info out of the 37362 files... The value for those 6 entries were `0`... I double checked with one of those 6 MINC files and it was indeed set to `0`. I checked another MINC file without the entry in `parameter_file` and indeed, this field was not in the MINC file. I also looked into the DICOMs of those files and could not find any entries for 0018x1149 in either type of files. So weird...

I think we can safely remove the FOV entries from `mri_protocol` given this and the fact that the code itself does not check for the FOV when determining the scan type.